php/hyperf/composer.json

```diff
@@ -7,13 +7,19 @@
     "hyperf/dispatcher": "*",
     "hyperf/event": "*",
     "hyperf/exception-handler": "*",
-    "hyperf/framework": "~2.1.0",
+    "hyperf/framework": "~2.2.0",
     "hyperf/http-server": "*",
     "hyperf/utils": "*"
   },
   "autoload": {
     "psr-4": {
       "App\\": "app/"
     }
+  },
+  "minimum-stability": "dev",
+  "prefer-stable": true,
+  "config": {
+    "optimize-autoloader": true,
+    "sort-packages": true
```

Comment:
no need, done in the dockerfile https://github.com/the-benchmarker/web-frameworks/blob/e685546fb7cae7d926753bbd067c6cb66c1f3ebb/php/Dockerfile#L39